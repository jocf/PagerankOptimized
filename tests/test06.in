3
0.85
3
A
B
C
2
A B
B C
