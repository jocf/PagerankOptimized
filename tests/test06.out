A 0.0500
B 0.0925
C 0.1286
